src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -207,6 +307,8 @@ THE SOFTWARE.
                         this.$props.data.joomlaRules.enabled = true;
                         this.$props.data.joomlaRules.computed = this.$props.data.joomlaRules.value;
                     } else {
+                        this.$props.data.phpServer.enabled = false;
+                        this.$props.data.phpBackupServer.enabled = false;
```

Comment:
```suggestion
                        this.$props.data.phpServer.enabled = false;
                        this.$props.data.phpServer.computed = '';
                        this.$props.data.phpBackupServer.enabled = false;
                        this.$props.data.phpBackupServer.computed = '';
```
Just to be safe and consistent with how we normally disable stuff, also set their computed values to be empty strings.